test/src/test/java/lib/form/RepeatablePropertyTest.java

```diff
@@ -69,6 +76,32 @@ public void testFieldNotNullWithDefaultIgnoresDefaults() throws Exception {
         ));
         assertFormContents(VIEW_WITH_DEFAULT, testRepeatable);
     }
+
+    @Issue("JENKINS-37599")
+    public void testNestedRepeatableProperty() throws Exception {
+        testRepeatableContainer = Collections.emptyList();
+        // minimum="1" is set for the upper one,
+        // the form should be:
+        // * 1 ExcitingObjectCotainer
+        // * no ExcitingObject
+        final HtmlForm form = getForm("nested");
+        List<HtmlTextInput> containerNameInputs = form.getElementsByAttribute("input", "type", "text");
+        CollectionUtils.filter(containerNameInputs, new Predicate<HtmlTextInput>() {
+            @Override
+            public boolean evaluate(HtmlTextInput input) {
+                return input.getNameAttribute().endsWith(".containerName");
+            }
+        });
+        List<HtmlTextInput> greatPropertyInputs = form.getElementsByAttribute("input", "type", "text");
+        CollectionUtils.filter(greatPropertyInputs, new Predicate<HtmlTextInput>() {
+            @Override
+            public boolean evaluate(HtmlTextInput input) {
+                return input.getNameAttribute().endsWith(".greatProperty");
+            }
+        });
+        assertEquals(1, containerNameInputs.size());
+        assertEquals(0, greatPropertyInputs.size());
```

Comment:
Without this fix, this fails with "junit.framework.AssertionFailedError: expected:<0> but was:<1>" as `minimum="1"` in 
RepeatablePropertyTest/nested.jelly is unexpectedly propagated to ExcitingObjectContainer/config.jelly .